src/node-lib/curriculum-api-2023/queries/pupilLessonOverview/pupilLessonOverview.query.ts

```diff
@@ -29,7 +30,11 @@ export const pupilLessonOverviewQuery =
       });
     }
 
+    const isLegacyLicense =
+      !lesson.lessonCohort || lesson.lessonCohort === LEGACY_COHORT;
```

Comment:
I think lessonCohort should never be undefined so the first part of this conditional is unnecessary.

Reply:
😞 the GQL types disagree, it can be `null` or `undefined`